scripts/junos/jinja_templates/revert_nonchannelized_port.j2

```diff
@@ -0,0 +1,16 @@
+{# variable 'port_name' should be like 'et-0/0/11' #}
+{# or 'et-0/0/12' or 'et-0/0/8' #}
+{# Also make sure that dictionary passed to template_vars #}
+{# contains keys that matches the variable names that are used here #}
+interfaces {
+            {{ port_name }} {
+    unit 0 {
+        family ethernet-switching {
+            vlan {
+                members default;
+            }
+            storm-control default;
```

Comment:
why are you setting storm-control only on a non-channelized port, why not in a channelized port?

Reply:
these are the only 2 differences in the .j2 files for channelized and non channelized ports. 

Also, can you add a description to each `.j2` file what it's doing and why it's doing? because it's not obvious to someone like me who doesn't know much about jinja templates.

Reply:
The settings in the file are what the administrators at Redhat wanted for their switch.
That was taken `as is`. Also that is the idea that they can set `any default state` for ports as they wish and HIL will comply with the requirements without having to know the reasons to do so. There can be settings that may affect the switch as a whole but it may have no effect on the operations that HIL administers.